src/installer/installer.go

```diff
@@ -364,7 +369,7 @@ func (i *installer) waitForControlPlane(ctx context.Context) error {
 	i.waitForBootkube(ctx)
```

Comment:
this will always return immidetely, no?

Reply:
This will return if bootkube service compelted.
In 4.6 it took some time (due to the etcd patch) but since 4.7 I think it returns quickly since bootkube finish before the 2 master nodes are ready.